api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -108,6 +115,22 @@ impl RpcServer for RpcServerImpl {
 			.await
 			.map(BrokerSwapDepositAddress::from)?)
 	}
+
+	async fn withdraw_fee_asset(
+		&self,
+		asset: Asset,
+		destination_address: String,
+	) -> RpcResult<String> {
+		Ok(self
+			.api
+			.broker_api()
+			.withdraw_fee_asset(
+				asset,
+				clean_foreign_chain_address(asset.into(), &destination_address)?,
+			)
+			.await
+			.map(|tx_hash| format!("{tx_hash:#x}"))?)
```

Comment:
As discussed, better to return the hash here and rely on Serialize rather than converting to a string explicitly.